sdk/core/core/test/curl_easy_perform_poc.c

```diff
@@ -88,27 +88,19 @@ int main() {
     return add_header_result;
   }
 
-  // *************************send GET
-  az_result const get_response = az_http_client_send_request(&hrb, &http_buf_response);
-  if (az_failed(get_response)) {
-    printf("Error after request key from KeyVault\n");
-    return get_response;
-  }
+  // *************************send GET using Pipeline
+  // TODO: Remove previous Request set-up and let Pipeline to set that up
+  az_result const get_response = az_http_pipeline_process(&hrb, &http_buf_response);
 
   if (az_succeeded(get_response)) {
     printf("Response is: \n%s", http_buf_response.begin);
   } else {
     printf("Error during running test\n");
```

Comment:
Could we create `az_log.h` with `#define AZ_LOG(...) do { printf(…); } while(0)

Reply:
https://github.com/Azure/azure-sdk-for-c/issues/172